parachain-template/polkadot-launch/config.json

```diff
@@ -17,7 +17,7 @@
 	},
 	"parachains": [
 		{
-			"bin": "../target/release/parachain-collator",
+			"bin": "../target/release/polkadot-parachain",
```

Comment:
```suggestion
			"bin": "../target/release/polkadot-template-node",
```

Reply:
`parachain-template-node`

Reply:
I think the rename as I did it is what we want. It was using the template but it was a mistake IMHO.